src/mobile/components/ModalWithdrawConfirmation/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { useIntl } from 'react-intl';
+import { Decimal } from '../../../components';
+import { Modal } from '../../components';
+
+interface ModalWithdrawConfirmationProps {
+    amount: string;
+    currency: string;
+    onSubmit: () => void;
+    onDismiss: () => void;
+    rid: string;
+    show: boolean;
+    precision: number;
+}
+
+const ModalWithdraw = (props: ModalWithdrawConfirmationProps) => {
+    const intl = useIntl();
+
+    const translate = (e: string) => {
+        return intl.formatMessage({id: e});
+    };
+
+    const renderHeader = () => {
+        return (
+            <div className="pg-exchange-modal-submit-header">
+                {translate('page.mobile.wallet.withdraw.modal.confirmation')}
+            </div>
+        );
+    };
+
+    const renderBody = () => {
+        const { amount, currency, precision, rid } = props;
+        const formattedCurrency = currency.toUpperCase();
```

Comment:
we use this variable just once, no need for creating it, use **currency.toUpperCase()** in the code